File: fizzbuzz.py
def fizzbuzz(n):
    print("--------------------")
    for i in range(1,n+1):
        if i % 15 ==0:
            print("fizzbuzz")
        elif i % 5 ==0:
            print("buzz")
        elif i % 3 == 0:
            print("fizz")
        else:
            print(i)

print("数字を入力してください")
number= int(input())
fizzbuzz(number)

File: fizzbuzz.rb
def fizzbuzz(n)
  puts "-----------------------"
  (1..n).each do |i|
    if i % 15 ==0
      puts "fizz_buzz"
    elsif i % 5 ==0
      puts "buzz"
    elsif i % 3 == 0
      puts "fizz"
    else
      puts i
    end
  end
end

puts "数字を入力してください"
number = gets.to_i
fizzbuzz(number)

